src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,34 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final char FIRST_CHARACTER_VALUE = 'w';
+    private static final int FIRST_CHARACTER = 0;
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> resultList = new ArrayList<>();
+        List<String> strings = null;
+
+        try {
+            strings = Files.readAllLines(Paths.get(fileName));
+        } catch (IOException e) {
+            System.out.println("There are no such file.");
+        }
+        for (String string : strings) {
+            String[] wordArray = string.split(" ");
+            for (String word : wordArray) {
+                if (word.toLowerCase().charAt(FIRST_CHARACTER) == FIRST_CHARACTER_VALUE) {
```

Comment:
there is a predefined method that checks if a String starts with a certain value